SB2018/2018-04-03.md

```diff
@@ -0,0 +1,13 @@
+---
+to: All Teams
+subject: Media Consent Forms
+---
+
+Hi all,
+
+In order for anyone to attend the competition, a 'Media Consent' Form must be filled out. If students are over 16, they can fill out the forms themselves; however, if students are under 16, the form must be signed by their parent/guardian.
+
+These forms must be filled out by all attendees and brought to the competition in order to gain entry.
+Any additional guests can sign a form on the door as long as they're 16 or over.
```

Comment:
Perhaps phrase this to cope with under-16 guests too?
> We'll have some forms on the door for additional guests, though we encourage that everyone completes them ahead of time.

Reply:
Hrm, given my proposed changed to the following paragraph it might make sense to move this line below that?

I'm not sure where that leaves the line "These forms must be..." though.

Reply:
fixed in8f9e501dc2fd51aeb106b0311df66208e53ffa1f